### DISCUSSION TITLE
Some files and folders are not added to the exe package which is defined in the assets section

### DISCUSSION THREAD

**Discussion by mega6453** [yao-pkg/pkg#48] [Q&A]
Created exe package using pkg with below package.json.. used fs.copy to copy the appium-uiautomator2-driver folder during runtime. It's copying files but not all. Some files and folders are missing. I feel while adding itself, it missed some files and folders. 

How to make sure all the files and folders are added to the package which is mentioned in the assets section?

{
  "name": "appium",
  "version": "2.5.1",
  "description": "Appium Server",
  "main": "index.js",
  "scripts": {
    "test": "appium"
  },
  "bin": {
    "appium": "./index.js"
  },
  "author": "Meganathan C",
  "license": "ISC",
  "dependencies": {
    "appium": "^2.5.1",
    "fs-extra": "^11.2.0",
    "pkg": "^5.8.1",
    "user-home": "^3.0.0"
  },
  "pkg": {
    "targets": [
      "node16-win-x64"
    ],
    "assets": [
      "./node_modules/appium-uiautomator2-driver/**",
      "./node_modules/appium-xcuitest-driver/**",
      "./node_modules/fs-extra/**",
      "./node_modules/axios/dist/node/axios.cjs",
      "./node_modules/sharp/build/**",
      "./node_modules/sharp/vendor/**"
    ],
    "outputPath": "dist"
  }
}

**Comment by mega6453** [comment]
@robertsLando

**Reply by robertsLando** [reply]
Try to run pkg using `-d` option to see if you see them there. Also when you run the packaged app you can use the `DEBUG_PKG=1` env var to see what is loaded inside the virtual filesystem. More info here: https://github.com/yao-pkg/pkg#advanced
